### ISSUE TITLE
Heists. 

### ISSUE THREAD

**Issue by Zorral** [dankgrinder/dankgrinder#33]
Do you think there could be a way of adding a heist sniper and automatically join them? Or giveaways. Thanks btw for the instant response on my last post and keep up the good work!

**Comment by 5Haze** [comment]
this is a great idea.

**Comment by ihavezerohealth** [comment]
Have you got an idea for how exactly to recognise these heists?

**Comment by PandoricGalaxy** [comment]
Yes it's the same way like the event how it recognises it the same way to recognise a heist and join it after a delay of 5-10 seconds
